config/ignition_edifice_debian_buster.yaml

```
@@ -16,30 +16,30 @@ Package (% =libignition-common4-graphics) |\
 Package (% =libignition-common4-graphics-dev) |\
 Package (% =libignition-common4-profiler) |\
 Package (% =libignition-common4-profiler-dev)), \
-$Version (% 4.0.0-1~*)) |\
-(Package (% =ignition-edifice), $Version (% 1.0.0-1~*)) |\
+$Version (% 4.4.0-*)) |\
+(Package (% =ignition-edifice), $Version (% 1.0.2*)) |\
```

Looks like one was missed.

```suggestion
(Package (% =ignition-edifice), $Version (% 1.0.2-*)) |\
```